docs/_static/about.inc

```diff
@@ -3,7 +3,8 @@
 About
 =====
 
-The **nimi-python** repository generates Python bindings (Application Programming Interface) for interacting with the Modular Instrument drivers. Currently, the following drivers are supported:
+The **nimi-python** repository generates Python bindings (Application Programming Interface) for interacting with the Modular Instrument drivers. Currently,
```

Comment:
This text is duplicated from README.rst.
We don't single source this text, right? I think that's fine just looking for confirmation.

Reply:
This is the single source for all the different locations including the global README.rst